src/python/pysss_nss_idmap.c

```diff
@@ -302,7 +317,9 @@ static int do_lookup(enum lookup_type type, PyObject *py_result,
         return do_getnamebysid(py_result, py_inp);
         break;
     case SIDBYID:
-        return do_getsidbyid(py_result, py_inp);
+    case SIDBYUID:
+    case SIDBYGID:
+        return do_getsidbyid(type, py_result, py_inp);
```

Comment:
Only when I'm looking at this code did I notice the switch uses:
```
return foo();
break;
```

But since also the other case statements use the same, I think it would be better to fix this in a follow-up patch.